src/components/shop-app.js

```diff
@@ -203,14 +200,16 @@ class ShopApp extends Element {
 
     <shop-analytics key="UA-39334307-16"></shop-analytics>
 
-    <iron-media-query query="max-width: 767px" query-matches="{{smallScreen}}"></iron-media-query>
-
     <app-header role="navigation" id="header" effects="waterfall" condenses="" reveals="">
       <app-toolbar>
         <div class="left-bar-item">
-          <paper-icon-button class="menu-btn" icon="menu" on-click="_toggleDrawer" aria-label="Categories">
+          <paper-icon-button class="menu-btn" icon="menu"
+              on-click="${_ => this.drawerOpened = true}"
+              aria-label="Categories"
+              hidden="${page === 'detail'}">
```

Comment:
What about also checking for `_smallScreen`, e.g. `hidden="${!_smallScreen || page === 'detail'}"`? After that we can remove these: [style1](https://github.com/Polymer/shop/blob/f96e406e8b302d1dcf3535612036f3361d52b239/src/components/shop-app.js#L87-L89), [style2](https://github.com/Polymer/shop/blob/f96e406e8b302d1dcf3535612036f3361d52b239/src/components/shop-app.js#L190-L196)